catalog/gke/cluster/README.md

```diff
@@ -0,0 +1,31 @@
+# GKE Cluster Package
+
+A kpt package to configure a GKE cluster.
+
+## Resources
+
+- [cluster.yaml](/catalog/gke/cluster/cluster.yaml)
+  - [ContainerCluster](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containercluster) to configure a GKE cluster.
+- [container-api.yaml](/catalog/gke/cluster/container-api.yaml)
+  - [Service](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) to enable the container.googleapis.com API.
+
+## Setters
+
+Setters are inherited by sub-packages.
+
+```
+$ kpt cfg list-setters .
```

Comment:
I don't think we have a replacement in v1 yet
https://github.com/GoogleContainerTools/kpt/issues/2097

Reply:
Yeah, the readmes maybe don't make much sense without a replacement.
I could just remove them and link to the setters.yaml tho.

Reply:
Updated using https://github.com/GoogleCloudPlatform/blueprints/pull/42